frontend/main-app/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Heading, Link } from "@digdir/design-system-react";
+import { ClipboardCheckmarkFillIcon } from "@navikt/aksel-icons";
+import classes from "./Header.module.css";
+
+export const Header = (): React.JSX.Element => {
+  return (
+    <header className={classes.header}>
+      <Link href="/" className={classes.headerLink}>
+        <Heading level={1} href="/">
+          Form Factory
+        </Heading>
+        <ClipboardCheckmarkFillIcon fontSize="3rem" className={classes.headerIcon} />
```

Comment:
I think it might be better to move the ```fontSize``` into the CSS class ```headerIcon```. As the CSS class is already there, it might be more structured to have all styling at the same place 😄 
What do you think?

Reply:
Agreed 👍